### DISCUSSION TITLE
Email template link href with variables

### DISCUSSION THREAD

**Discussion by matusziak** [strapi/strapi#7609] [Legacy Topics]
Hi, I want to create an email template with hyperlink that contains variables, but I can't find anywhere in the docs how it could work. 
Example of what I tried:

`<a href="<%= URL %>?code=<%= TOKEN %>">Link</a>`

Any idea how to incorporate variables inside of the `href`?

**Comment by derrickmehaffy** [comment]
Hello @matusziak the only variables that can be used in the templates are the following: https://strapi.io/documentation/v3.x/plugins/users-permissions.html#templating-emails

**Reply by matusziak** [reply]
@derrickmehaffy that's ok, the variables load just fine if I use them for example inside of a `<p>.` But I don't know how to load them inside of the link `<a href="HERE">`.

**Reply by derrickmehaffy** [reply]
On that one I am not quite sure, let me poke @lauriejim or @alexandrebodin about this

**Reply by matusziak** [reply]
@derrickmehaffy Hi, any update on this? :-)

**Reply by derrickmehaffy** [reply]
@matusziak Alex was out of the office for the past 2 weeks, he should be back on monday.

**Reply by alexandrebodin** [reply]
Hum the a tag should work. The href format might be wrong. have you tried with a dumb http url ?
